host-bmc/host_pdr_handler.cpp

```diff
@@ -760,6 +761,12 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
                     updateContainerId<pldm_numeric_effecter_value_pdr>(
                         entityTree, pdr);
                 }
+                else if (pdrHdr->type == PLDM_ENTITY_AUXILIARY_NAMES_PDR)
+                {
+                    pdrTerminusHandle = 0xFFFF;
```

Comment:
why was this needed ?

Reply:
There is no specific terminus handle for PLDM_ENTITY_AUXILIARY_NAMES_PDR. So I put it as a default value instead of 0.